lib/ref.js

```diff
@@ -75,10 +75,11 @@ $Ref.prototype.get = function (path, options) {
  *
  * @param {string} path - The full path being resolved, optionally with a JSON pointer in the hash
  * @param {$RefParserOptions} options
+ * @param {string} [originalPath] - The original, unmodified path being resolved
  * @returns {Pointer}
  */
-$Ref.prototype.resolve = function (path, options) {
-  var pointer = new Pointer(this, path);
+$Ref.prototype.resolve = function (path, options, originalPath) {
```

Comment:
I don't see anywhere where this method is called with the new parameter.  Am I missing something?  Or is the parameter always `undefined`?  If the latter, then we can remove the parameter from here and from the `Pointer` constructor.

Reply:
it's used here https://github.com/BigstickCarpet/json-schema-ref-parser/pull/75/files#diff-37de81b33471a7915002750868466e52R157

Reply:
thanks.  Not sure how I missed that